include/pybind11/numpy.h

```diff
@@ -790,6 +795,18 @@ class array : public buffer {
         if (isinstance<array>(new_array)) { *this = std::move(new_array); }
     }
 
+    array reshape(ShapeContainer new_shape) {
+        detail::npy_api::PyArray_Dims d
+            = {reinterpret_cast<Py_intptr_t *>(new_shape->data()), int(new_shape->size())};
+        // try to reshape, set ordering param to 0 cause it's not used anyway
```

Comment:
I think it's best to remove this comment, and add a comment just above the `array reshape(ShapeContainer new_shape) {` line, e.g.
```
// Optional `order` parameter omitted here, to be added as needed.
```